internal/sherlock/sherlock.go

```diff
@@ -0,0 +1,62 @@
+package sherlock
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/broadinstitute/sherlock/internal/db"
+	"github.com/spf13/viper"
+)
+
+// package level variable holding a viper instance that will manage sherlock's config
+var (
+	Config = viper.New()
```

Comment:
Just curious, why is viper being wrapped?

Reply:
It's not really being wrapped unless I'm misunderstanding you. It's storing a viper instance as a package level variable. This is mainly done so that all the config initialization can happen in the sherlock package `init` function, which in turn is guaranteed to run before `main`. This is mainly to ensure the config is fully read in and parsed before anything will try to access it.